src/gui/DatabaseWidget.cpp

```diff
@@ -755,7 +755,7 @@ void DatabaseWidget::entryActivationSignalReceived(Entry* entry, EntryModel::Mod
 void DatabaseWidget::switchToEntryEdit()
 {
     Entry* entry = m_entryView->currentEntry();
-    Q_ASSERT(entry);
+    
```

Comment:
Asked myself the same question, especially because there is an if (!entry) return right after it which handles a non-existing entry much better and without crashing the program.